### DISCUSSION TITLE
How to set customized store path or file?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [milvus-io/milvus#26336] [Q&A and General discussion]
Just wonder is there a way to set customized data store file or path in python api?

Also, how to dump data easily with command line ?

**Comment by yhmo** [comment]
Sdk cannot change the store path of server.
Milvus storage depends on S3/MinIo: https://milvus.io/docs/deploy_s3.md

**Comment by lucasjinreal** [comment]
@yhmo  am using lite, than which folder it stores by default?
